crates/alerter/src/main.rs

```diff
@@ -122,10 +122,7 @@ impl ZeroExApi {
 
         tracing::debug!(url = url.as_str(), ?response, "0x");
 
-        Ok(match order.kind {
-            OrderKind::Buy => order.sell_amount >= response.sell_amount,
-            OrderKind::Sell => order.buy_amount <= response.buy_amount,
-        })
+        Ok(response.sell_amount <= order.sell_amount && response.buy_amount >= order.buy_amount)
```

Comment:
Should we be even more strict and:
```suggestion
        Ok(match order.kind {
            OrderKind::Buy => order.buy_amount == response.buy_amount && order.sell_amount >= response.sell_amount,
            OrderKind::Sell => order.buy_amount <= response.buy_amount && order.sell_amount == response.sell_amount,
        })
```

I'm not sure how our DEXagg solvers handle these cases for fill-or-kill orders.

Reply:
I think it is not needed because for example for a sell order if you can still fill it with less sell amount then the order is still matched so the equality check is overly strict.